ranger/config/rifle.conf

```diff
@@ -264,5 +264,14 @@ label wallpaper, number 14, mime ^image, has feh, X = feh --bg-fill "$1"
 label editor, !mime ^text, !ext xml|json|csv|tex|py|pl|rb|js|sh|php  = ${VISUAL:-$EDITOR} -- "$@"
 label pager,  !mime ^text, !ext xml|json|csv|tex|py|pl|rb|js|sh|php  = "$PAGER" -- "$@"
 
-# The very last action, so that it's never triggered accidentally, is to execute a program:
+
+######################################################################
+# The actions below are left so low down in this file on purpose, so #
+# they are never triggered accidentally.                             #
+######################################################################
+
+# Execute a file as program/script.
 mime application/x-executable = "$1"
+
+# Move the file to trash using trash-cli.
+has trash-put = trash-put -- "$@"
```

Comment:
I'd also add `mv "$@" "$XDG_DATA_DIR"/ranger-trash` (or whatever xdg dir is approriate).

Reply:
I don't think we need a fallback here. `rm` is perfectly fine if use doesn't want/need `trash-cli`.